### PR TITLE
Changed rulesDirectory entry of tslint-microsoft-contrib to be peer...

### LINKED ISSUE #30: Can't find rules of tslint-microsoft-contrib when installed as peer.
It seems like tslint-clean-code can't find the tslint-microsoft-contrib rules directory, when it's installed as a peer-dependency.

Changing tslint.json line 3 from "node_modules/tslint-microsoft-contrib" to "tslint-microsoft-contrib" fixes that, but not sure if it would then still find it inside of node_modules.

### PR DESCRIPTION
**Pull request by IZEDx**
...friendly. Fixes #30

### CODE REVIEW

**Comment by jgornick** [comment]
Let's get this in :) Easy win for those of us who also use tslint-microsoft-contrib in our projects.

File: tslint.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "rulesDirectory": [
-    "node_modules/tslint-microsoft-contrib",
```
Comment (Nimelrian) [inline tslint.json]: Note that running tslint will still error when `tslint-microsoft-contrib` is not installed. It is only specified as devDependency in package.json. Consider making it a peerDependency or even a full dependency.

Comment (Glavin001) [reply tslint.json]: Agreed, `peerDependency` makes sense.

**Comment by Glavin001** [comment]
AppVeyor build is currently failing: https://ci.appveyor.com/project/Glavin001/tslint-clean-code/build/1.0.43#L332

```
Running "tslint:prod" (tslint) task
Warning: Could not find custom rule directory: C:\projects\tslint-clean-code\tslint-microsoft-contrib Use --force to continue.
```

And it appears Travis CI decided to skip testing:
image

**Comment by Glavin001** [comment]
Here's my PR: https://github.com/Glavin001/tslint-clean-code/pull/34

Please take a look and 👍 and/or comment! I will wait for some feedback before merging and publishing.

**Comment by Glavin001** [comment]
Merged https://github.com/Glavin001/tslint-clean-code/pull/34 instead.
Thank you for contributing and bringing this to my attention!

**Pull request closed without merging**